graphql/project_settings_resolver.go

```diff
@@ -21,11 +22,16 @@ func (r *projectSettingsResolver) Aliases(ctx context.Context, obj *restModel.AP
 
 // GithubWebhooksEnabled is the resolver for the githubWebhooksEnabled field.
 func (r *projectSettingsResolver) GithubWebhooksEnabled(ctx context.Context, obj *restModel.APIProjectSettings) (bool, error) {
-	hook, err := model.FindGithubHook(utility.FromStringPtr(obj.ProjectRef.Owner), utility.FromStringPtr(obj.ProjectRef.Repo))
-	if err != nil {
-		return false, InternalServerError.Send(ctx, fmt.Sprintf("Database error finding github hook for project '%s': %s", *obj.ProjectRef.Id, err.Error()))
-	}
-	return hook != nil, nil
+	owner := utility.FromStringPtr(obj.ProjectRef.Owner)
+	repo := utility.FromStringPtr(obj.ProjectRef.Repo)
+	hasApp, err := evergreen.GetEnvironment().Settings().HasGitHubApp(ctx, owner, repo, nil)
+	grip.Error(message.WrapError(err, message.Fields{
```

Comment:
does not error as it breaks the project settings page but I still had to return error because I believe that's just what GraphQL resolvers expect.